docs/cli_gettingstarted.md

```diff
@@ -27,9 +28,38 @@ git clone https://github.com/blockvigil/ethvigil-cli.git
 cd ethvigil-cli
 pip install -e .
 ```
-## Sign up with invite code
 
-Head to [EthVigil Beta](https://beta.ethvigil.com/?clisignup=true) to generate an invite code.
+## Setting up your account
+
+There are a few handy paths you can take to setup or initialize your account on the EthVigil CLI. Choose one of them from the following as it fits your scenario.
+
+### ✅ Reuse your in-browser local wallet settings (No Metamask involved)
```

Comment:
We could probably collapse this to skip to init step? New users will be in majority and we want to get them onboarded ASAP.

Reply:
Reversing the order of the sections for now. Let's revisit this when we upgrade to Docusaurus v2.